waffle.json

```diff
@@ -7,7 +7,7 @@
     "evmVersion": "london",
     "optimizer": {
       "enabled": true,
-      "runs": 200
+      "runs": 10
```

Comment:
Because we already hit the contract size limit
```
Warning: Contract code size exceeds 24576 bytes (a limit introduced in Spurious Dragon). 
This contract may not be deployable on mainnet. 
Consider enabling the optimizer (with a low "runs" value!), 
turning off revert strings, or using libraries.
```

Reply:
Actually, I'd love to turn off revert strings if that's possible. That sounds lovely.

Reply:
After removing chainId whitelisting I was able to revert runs back to `200`